website/www/site/assets/scss/_navbar-desktop.scss

```diff
@@ -166,6 +166,12 @@
   
```

Comment:
General comment: could you make sure that items in navbar stick to the left on non-standard resolutions? It looks a bit awkward here
![Uploading image.png…]()

Reply:
Yes you are right, I will fix it